Sources/Apollo/GraphQLQueryWatcher.swift

```diff
@@ -78,9 +78,15 @@ public final class GraphQLQueryWatcher<Query: GraphQLQuery>: Cancellable, Apollo
 private extension CachePolicy {
   var alwaysIncludesServerFetch: Bool {
     switch self {
-    case .fetchIgnoringCacheCompletely, .fetchIgnoringCacheData, .returnCacheDataAndFetch:
+    case .fetchIgnoringCacheCompletely:
       return true
-    case .returnCacheDataDontFetch, .returnCacheDataElseFetch:
+    case .fetchIgnoringCacheData:
+      return true
+    case .returnCacheDataAndFetch:
+      return true
+    case .returnCacheDataDontFetch:
+      return false
+    case .returnCacheDataElseFetch:
```

Comment:
Sorry, should have been clearer here: I meant more: 

```swift
case .fetchIgnoringCacheCompletely,
     .fetchIgnoringCacheData,
     .returnCacheAndFetch:
  return true
``` 
etc

Reply:
Oh ok, I was actually wondering. Give me a sec!